in_toto/in_toto_sign.py

```diff
@@ -302,12 +302,12 @@ def main():
     if ((args.key != None and len(args.key) > 1) or
         (args.gpg != None and len(args.gpg) > 1)):
       parser.print_help()
-      parser.exit(2, "too many arguments: {} Hence signing Link metadata"
+      parser.error("too many arguments: {} Hence signing Link metadata"
           " with multiple keys is not allowed.".format(link_error_message))
 
     if args.append:
       parser.print_help()
-      parser.exit(2, "wrong arguments: {}. Hence adding signatures to"
+      parser.error("wrong arguments: {}. Hence adding signatures to"
```

Comment:
So the entire error message is:

```
("Link metadata is associated with a single functionary and is usually namespaced accordingly: '<name>.<keyid>.link'.")
Hence signing Link metadata with multiple keys is not allowed.
```

What if a functionary wants to use multiple keys?

Reply:
For instance, one functionary wants the ability to sign with one of two keys under their control.  One key is stored on their laptop, and a different, and also trusted, key is on their Yubikey.

Reply:
This behavior  was already topic for discussion in https://github.com/in-toto/in-toto/pull/125 :)

The specification currently does not explicitly prohibit signing one link with multiple keys, however due to the naming convention `<related step name>.<SIGNING KEY PREFIX>.link` and the way final product verification will load links for steps, it does make sense and that's why this utility prohibits it.

Note that this doesn't mean that a functionary can't have multiple authorized keys to sign a link. All the public keys that a link can be signed with need to be embedded in the layout's `keys` section, and in to authorize them for a specific step, the keyid's must be in the step's `pubkeys` list.